src/main/java/seedu/address/storage/Pair.java

```diff
@@ -14,6 +14,15 @@ public class Pair <K, V> implements Serializable {
     private final K key;
     private final V value;
 
+    /**
+     * Fulfils the role of a dummy constructor for JSON file reading compatibility.
+     * I don't know why this is needed. DO NOT USE IN CODE.
+     */
+    public Pair() {
```

Comment:
Since we do not want the Pair Dummy Constructor to be called, would it be better to make this Constructor private?